### PR TITLE
CB-10584 Splashscreen plugin crashes the app on windows 10 when built…

### PR DESCRIPTION
**Pull request by daserge**
… with browserify

[Jira issue](https://issues.apache.org/jira/browse/CB-10584)

### CODE REVIEW

**Comment by daserge** [comment]
It was taking the old version from `legacy_exec` so `base.js` was not found.

**Comment by daserge** [comment]
@vladimir-kotikov, could you please review?

File: cordova-lib/src/plugman/browserify.js [hunk 1]
```diff
@@ -106,7 +106,7 @@ module.exports = function doBrowserify (project, platformApi, pluginInfoProvider
         commitId = cId;
         return getPlatformVersion(commitId, platformApi.root);
     }).then(function(platformVersion){
-        var libraryRelease = bundle(platform, false, commitId, platformVersion);
+        var libraryRelease = bundle(platform, false, commitId, platformVersion, wwwDir);
```
Comment (vladimir-kotikov) [inline cordova-lib/src/plugman/browserify.js]: Shouldn't we use `platform_www` directory here instead of `<platform>/www`?

**Comment by daserge** [comment]
Added [CB-10616 Cover browserify scenario with tests](https://issues.apache.org/jira/browse/CB-10616).

**Pull request merged**